llama-index-integrations/storage/kvstore/llama-index-storage-kvstore-azure/llama_index/storage/kvstore/azure/base.py

```diff
@@ -185,7 +185,8 @@ def put(
         table_name = (
             DEFAULT_COLLECTION if not collection else sanitize_table_name(collection)
         )
-        self._table_client.create_table_if_not_exists(table_name).upsert_entity(
+        table_client = self._table_client.create_table_if_not_exists(table_name)
```

Comment:
unfortunately, it looks like `self._table_client` should be `self._table_service_client`